src/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  font-family: Roboto, Arial, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-image {
+  padding: 16px 0;
+  margin: 0 auto;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+  object-fit: fill;
+}
+
+.product-description {
+  padding: 18px 0 4px;
+}
+
+.product-description__link {
+  font-size: 12px;
+  line-height: 12px;
+  font-weight: 500;
+  color: #060b35;
+  text-decoration: none;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
```

Comment:
Move all repeatable colors to CSS variables in `:root` selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)